plenum/common/util.py

```diff
@@ -585,6 +585,6 @@ def isValidEndpoint(endpoint):
     ip, port = endpoint.split(':')
     try:
         ipaddress.ip_address(ip)
-        return port.isdigit()
+        return port.isdigit() and int(port) in range(1, 65535)
     except Exception:
```

Comment:
We shouldn't just be checking for Exception. This masks legitimate exceptions that should flow through.